### DISCUSSION TITLE
sysprefs_wifi_disable and os_filevault_authorized_users not in *_compliance.sh

### DISCUSSION THREAD

**Discussion by erefneb** [usnistgov/macos_security#96] [General]
I've noticed that the rules sysprefs_wifi_disable and os_filevault_authorized_users do not make it into the *_compliance.sh
I've tested this with guidance for stig and all_rules.
Unfortunately nothing in the rules' yaml seems wrong or sticks out to me.

**Comment by robertgendler** [comment]
Rules that have -manual tag won't get added to the compliance script to check or remediate. We decided for 1 reason or another that these should not have or cannot have automated remediation.
